lib/util.js

```diff
@@ -85,31 +85,36 @@ exports.truthy = function(arg) {
   return !!arg;
 };
 
-exports.nextTick = function(callback) {
-  if (typeof process !== 'undefined' && process.nextTick) {
-    return process.nextTick.apply(null, arguments);
-  }
-
-  var args = [];
-  for (var i = 1; i < arguments.length; i++) {
-    args[i - 1] = arguments[i];
-  }
+if (typeof process !== 'undefined' && typeof process.nextTick === 'function') {
+  exports.nextTick = process.nextTick.apply(null, arguments);
```

Comment:
I think this can just be:
```
  exports.nextTick = process.nextTick;
```

`nextTick.apply` immediately invokes nextTick at module load time, which isn't quite what we want. Plus `arguments` won't be defined at module level.